### PR TITLE
Add Count support

### PR DESCRIPTION
**Pull request by shubham-cmyk**
Fix : https://github.com/kyverno/chainsaw/issues/97

### CODE REVIEW

**Comment by shubham-cmyk** [comment]
@eddycharly PTAL

**Comment by eddycharly** [comment]
Add a log in the test command too

File: pkg/apis/v1alpha1/configuration_spec.go [hunk 1]
```diff
@@ -66,4 +66,11 @@ type ConfigurationSpec struct {
 	// IncludeTestRegex is used to include tests based on a regular expression.
 	// +optional
 	IncludeTestRegex string `json:"includeTestRegex,omitempty"`
+
+	// RepeatCount indicates how many times the tests should be executed.
+	// +kubebuilder:default:=1
```
[the remark this answers is not in the document]
Comment (shubham-cmyk) [reply pkg/apis/v1alpha1/configuration_spec.go:71]: Yes i think int default is 0 which is wrong.

[the remark this answers is not in the document]
Comment (eddycharly) [reply pkg/apis/v1alpha1/configuration_spec.go:71]: default for a pointer is `nil`

[the remark this answers is not in the document]
Comment (shubham-cmyk) [reply pkg/apis/v1alpha1/configuration_spec.go]: fixed

File: pkg/commands/test/command.go [hunk 2]
```diff
@@ -156,6 +163,7 @@ func Command() *cobra.Command {
 	cmd.Flags().BoolVar(&options.skipDelete, "skip-delete", false, "If set, do not delete the resources after running the tests.")
 	cmd.Flags().BoolVar(&options.failFast, "stop-on-first-failure", false, "Stop the test upon encountering the first failure.")
 	cmd.Flags().IntVar(&options.parallel, "parallel", 8, "The maximum number of tests to run at once.")
+	cmd.Flags().IntVar(options.repeatCount, "repeatCount", *options.repeatCount, "Number of times to repeat each test.")
```
Comment (eddycharly) [inline pkg/commands/test/command.go:166]: flag should be `repeat-count`

File: pkg/runner/runner.go [hunk 3]
```diff
@@ -55,17 +55,26 @@ func Run(cfg *rest.Config, config v1alpha1.ConfigurationSpec, tests ...discovery
...
-	}
-	internalTest := []testing.InternalTest{{
-		Name: "chainsaw",
-		F:    run,
-	}}
-	var testDeps testDeps
-	m := testing.MainStart(&testDeps, internalTest, nil, nil, nil)
-	return m.Run(), &summary, nil
+
+	var err error
```
Comment (eddycharly) [inline pkg/runner/runner.go:59]: error handling seems wrong here

File: pkg/runner/runner.go [hunk 4]
```diff
@@ -55,17 +55,26 @@ func Run(cfg *rest.Config, config v1alpha1.ConfigurationSpec, tests ...discovery
...
+	for i := 0; i < *config.RepeatCount; i++ {
```
Comment (eddycharly) [inline pkg/runner/runner.go:60]: repeat count can be nil

**Pull request closed without merging**